custom/icds_reports/static/js/directives/month-filter/month-filter.directive.js

```diff
@@ -108,6 +115,23 @@ function MonthFilterController($scope, $location, $uibModal, storageService) {
             $scope.$emit('filtersChange');
         });
     };
+
+    var init = function() {
+            var month = $location.search()['month'];
+            var year = $location.search()['year'];
+            var display_modal = true;
+
+            if(year > 2019 || year==2019&&month>1){
```

Comment:
add white spaces around operators

add brackets to the condition, will be more readable in future